data_validation/cli_tools.py

```diff
@@ -488,21 +488,24 @@ def _configure_schema_parser(schema_parser):
 
 def _configure_custom_query_parser(custom_query_parser):
     """Configure arguments to run custom-query validations."""
-    _add_common_arguments(custom_query_parser)
+    _add_common_arguments(custom_query_parser, consts.CUSTOM_QUERY)
```

Comment:
See below; don't need to pass in the query_type.

Reply:
Fixed [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/7667b5f603a4796bb96360ec48b7c66a9611179e)